apps/andi/lib/andi_web/live/dataset_live_view.ex

```diff
@@ -57,4 +83,19 @@ defmodule AndiWeb.DatasetLiveView do
   defp search_contains?(str, search_str) do
     String.downcase(str) =~ String.downcase(search_str)
   end
+
+  defp sort_by_dir(datasets, order_by, order_dir) do
+    case order_dir do
+      "asc" -> Enum.sort_by(datasets, fn dataset -> Map.get(dataset, order_by) end)
+      "desc" -> Enum.sort_by(datasets, fn dataset -> Map.get(dataset, order_by) end, &>=/2)
+      _ -> datasets
+    end
+  end
+
+  defp to_view_model(dataset) do
+    %{
+      "org-title" => dataset.business.orgTitle,
```

Comment:
is kabob case the norm here?  Normally we expect elixir vars to be snake case?

Reply:
These are the user facing ones. I'm open to changing it, or maybe mixing a little of both in?